test/e2e/kubectl/kubectl.go

```diff
@@ -2069,6 +2073,97 @@ metadata:
 			gomega.Expect(outBuiltIn).To(gomega.Equal(outStatusSubresource))
 		})
 	})
+
+	ginkgo.Describe("Kubectl rollout", func() {
```

Comment:
One ask, since you're adding new rollout tests, and we're trying to slim down the size of this file, can you move this test to a new file called `rollout.go` in this directory?